packages/access-client/src/agent.js

```diff
@@ -491,7 +755,7 @@ export class Agent {
    */
   async #waitForDelegation(opts) {
```

Comment:
We can make this a function(agent) since it doesn't rely on any private api afaict

Reply:
https://github.com/web3-storage/w3protocol/pull/563